sample/SampleHost/Program.cs

```diff
@@ -19,6 +21,24 @@ static void Main(string[] args)
                 config.UseDevelopmentSettings();
             }
 
+            // Build up a LoggerFactory to log to App Insights, but only if this key exists.
+            string instrumentationKey = Environment.GetEnvironmentVariable("ApplicationInsightsInstrumentationKey");
```

Comment:
Yes, this is just a sample to show people they can build up the logger however they want. In our Functions implementation (in another repo -- coming once this PR is done), we do the config automatically and I use APPINSIGHTS_INSTRUMENTATIONKEY.